### PR TITLE
new onboarding cheat sheet

### PR DESCRIPTION
**Pull request by dbradish-microsoft**

### CODE REVIEW

File: docs-ref-conceptual/cheat-sheet-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+## Learning Azure CLI
+
+There are several reference commands that provide interactive prompts to help you learn the Azure CLI.
+
+| Command | Description | Example
+|-|-|-|
+| [az interactive](interactive-azure-cli.md) | In a PowerShell console, get auto-completion, command descriptions, and examples as you type | See [Work in interactive mode](interactive-azure-cli.md) for more information.  Example: Type `az interactive` in your console window then <kbd>Enter</kbd>.
+| [az scenario](/cli/azure/scenario) | Get end to end (E2E) examples and execute them with walk-through prompts. | Type `az scenario guide "keywords of interest"` in your console then <kbd>Enter</kbd>.  The Azure CLI will return scenarios based on your keyword(s).  Example: `az scenario guide "virtual machines"`
+| [az next](/cli/azure/reference-index#az-next) | Get recommendations for the next Azure CLI command based on customer execution patterns.  You will be prompted to select a command path. | Type `az next` in your console then <kbd>Enter</kbd>.  Pick from 1 of 7 preset command paths.
```
Comment (daphnemamsft) [inline docs-ref-conceptual/cheat-sheet-onboarding.md:63]: "You will be prompted" should be changed to "You are prompted" for present tense

File: docs-ref-conceptual/cheat-sheet-onboarding.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+| [az find](/cli/azure/reference-index#az-find) | This is an AI robot that will return reference examples for keywords. | Type `az find "active subscription"` in your console then <kbd>Enter</kbd>.
```
Comment (daphnemamsft) [inline docs-ref-conceptual/cheat-sheet-onboarding.md:64]: Use present tense for "will returns"

File: docs-ref-conceptual/cheat-sheet-onboarding.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
...arding tips in this article?  Please use the [Feedback for this page](https://github.com/MicrosoftDocs/azure-docs-cli/issues/new?title=&body=%F0%9F%9A%A8%20Looks%20like%20you%20arrived%20from%20the%20internal%20review%20site%20%0A%F0%9F%9A%A8%20Do%20not%20enter%20Microsoft%20confidential%20information%20here%0A%0A%5BEnter%20feedback%20here%5D%0A%0A%0A---%0A%23%23%23%23%20Document%20Details%0A%0A%E2%9A%A0%20*Do%20not%20edit%20this%20section.%20It%20is%20required%20for%20learn.microsoft.com%20%E2%9E%9F%20GitHub%20issue%20linking.*%0A%0A*%20ID%3A%207e619320-8638-7beb-48ce-fa00a5d67779%0A*%20Version%20Independent%20ID%3A%20e5d2a07f-afd9-e985-abda-0840ac5f59b0%0A*%20Content%3A%20%5BAzure%20CLI%20onboarding%20cheat%20sheet%5D(https%3A%2F%2Freview.learn.microsoft.com%2Fen-us%2Fcli%2Fazure%2Fcheat-sheet-onboarding%3Fbranch%3Dpr-en-us-3776)%0A*%20Content%20Source%3A%20%5Bdocs-ref-conceptual%2Fcheat-sheet-onboarding.md%5D(https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fazure-docs-cli%2Fblob%2Fmain%2Fdocs-ref-conceptual%2Fcheat-sheet-onboarding.md)%0A*%20Service%3A%20**azure-cli**%0A*%20GitHub%20Login%3A%20%40dbradish-microsoft%0A*%20Microsoft%20Alias%3A%20**dbradish**) link below and let us know!
```
Comment (daphnemamsft) [inline docs-ref-conceptual/cheat-sheet-onboarding.md:79]: Remove "please"

Comment (daphnemamsft) [inline docs-ref-conceptual/cheat-sheet-onboarding.md:79]: For "link below" try to avoid using directional terms (acrolinx)

**Review by daphnemamsft — CHANGES_REQUESTED** [review]

**Comment by dbradish-microsoft** [comment]
All recommendations implemented.
Review received by @daphnemamsft (GitHub) and @ChaseCrum (via Teams)

**Pull request merged**